plugins/scripts/optimove.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>apiKey</key>
+        <string>$(API_KEY)</string>
```

Comment:
I think these values should be:

1) updated to be the optimove & mobile credentials (two keys)
2) the placeholder syntax that `renderTemplate` uses in inject-config (e.g. `{{OPTIMOVE_CREDENTIALS}}`)

Reply:
i can't compile to app with this syntax, it must be a valid plist

Reply:
Before the app is compiled, the inject-config script should run and translate the syntax by replacing the `{{}}` tokens with real values, making the syntax correct.

The script can be triggered with `cordova prepare` in the test app and the output should give some info about running the hook.

So, in our plugin sources, the plist should be a template (and it won't be valid), but in the app's Resources bundle, it should be a valid one because it was processed by the script.

Reply:
done